src/main/java/org/minions/devfund/miguelcalderon/CountCharacters.java

```diff
@@ -0,0 +1,34 @@
+package org.minions.devfund.miguelcalderon;
+
+
+/**
+ * Solution for "count the characters" kata.
+ *
+ * @version 1.0 18 Jul 2018  * @author
+ * Miguel Calderon
+ */
+
+public class CountCharacters {
+
+    /**
+     * Private constructor.
+     */
+    private CountCharacters() {
+    }
+
+    /**
+     * Counts the how many times char "e" appears in word.
+     */
+    public static void main(final String[] args) {
+        String word = "Martes de fuchitos";
+        char character = 'e';
+        int result = 0;
+
+        for (char c : word.toCharArray()) {
+            if (c == character) {
+                result++;
+            }
+        }
+        System.out.print(result);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)

Reply:
replaced by logger